.github/workflows/python-lib-release.yml

```diff
@@ -1,158 +1,11 @@
-          path: ${{ steps.build.outputs.artifact-path }}
-
-      - name: Delete release branch if deploy failed and fail
-        if: steps.build.outcome == 'failure'
-        run: |
-          git push origin -d release
-          echo "There was an error in the python build and publish commands above."
-          exit 1
-        shell: bash
-
-  create-release:
-    needs: [deploy]
-    runs-on: ubuntu-latest
-    container: zepben/pipeline-basic:5.0.0
-    steps:
-      - uses: actions/checkout@v2
-      - name: Merge and Tag
-        id: merge
-        env:
-          VERSION: ${{ needs.deploy.outputs.version }}
-        run: |
-          git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-          git fetch --all
-          git merge origin/release
-          git push origin ${GITHUB_REF/refs\/heads\//}
-          git tag "v$VERSION"
-          git push --tags
-          echo "::set-output name=tag::$(echo v$VERSION)"
-        shell: bash
-
-      - name: Fail
-        if: steps.merge.outcome == 'failure'
-        run: |
-          git push origin -d release
-          echo "There was an error in merging the branch. release branch was deleted."
-          exit 1
-        shell: bash
-
-      - name: Create Release
-        if: steps.merge.outcome == 'success'
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ steps.merge.outputs.tag }}
-          release_name: ${{ steps.merge.outputs.tag }}
-          body_path: changelog.md
-          draft: false
-          prerelease: false
-
-      - name: Download binary
-        uses: actions/download-artifact@v2
-        with:
-          name: ${{ needs.deploy.outputs.artifact }}
-          path: built-artifacts
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: built-artifacts/${{ needs.deploy.outputs.artifact }}
-          asset_name: ${{ needs.deploy.outputs.artifact }}
-          asset_content_type: application/octet-stream
-
-  update-version:
-    needs: create-release
-    runs-on: ubuntu-latest
-    container: zepben/pipeline-basic
-    env:
-      ZEPBEN_PYPI_USERNAME: ${{ secrets.NEXUS_USERNAME }}
-      ZEPBEN_PYPI_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
-      ZEPBEN_PYPI_REPO: ${{ secrets.ZEPBEN_PYPI_REPO }}
-      SLACK_NOTIFICATION: YES
-      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Update to next minor version
-        run: |
-          /scripts/update-version.sh --python --release setup.py changelog.md
-        shell: bash
+  run:
+    uses: zepben/.github/.github/workflows/python-lib-release.yml@main
+    with:
+      private: true
```

Comment:
this one is a public repo, so this should be false and licencecheck needs to be true as well.

Reply:
Done